docs/search-views/fulltext-search/concepts/primary-sort-order.md

```diff
@@ -1,15 +1,19 @@
 ---
-sidebar_position: 90
-title: Primary Sort Order
+sidebar_position: 20
+title: Understanding Primary Sort Order
 ---
 
-When you create a search view, you can choose a primary sort order for each uniquely named attribute, enabling better optimization for iterated C8QL queries that sort by one or more attributes. If the fields match the sorting directions, the search view can read data from the index without a sorting operation. 
```

Comment:
I think following is important and should get covered - `you can choose a primary sort order for each uniquely named attribute`